internal/server/proto/server.proto

```diff
@@ -320,6 +320,9 @@ message Variable {
 
   oneof value {
     string str = 2;
+    bool bool = 9;
+    int64 num = 10;
+    // Unimplemented
```

Comment:
is this a future todo? should you mark it `TODO`?

Reply:
This was a historic `TODO` I forgot to remove because I didn't mark it `TODO` so it didn't show up in my search 😅